crates/sui-e2e-tests/tests/validator_tx_finalizer_e2e_tests.rs

```diff
@@ -31,7 +31,7 @@ async fn test_validator_tx_finalizer_fastpath_tx() {
         .handle_transaction(tx, None)
         .await
         .unwrap();
-    tokio::time::sleep(Duration::from_secs(120)).await;
+    tokio::time::sleep(Duration::from_secs(180)).await;
```

Comment:
Given that the max back off here will be `6*10 + 60 = 120` I guess using a lower number would probably do , but we can go with that to be on the safe side. I would also suggest adding a small comment on why we are picking that number here.

Reply:
I made these test less aggressive, i.e. it actually forms a cert first, so that the wait time is under better control

Reply:
That seems even better 👍